### ISSUE TITLE
Ordered mode and channels support for RPC.

### ISSUE THREAD

**Issue by Malkverbena** [Faless/godot-enet-better#6]
Could the ordered mode and channels support for rpc be implemented?

**Comment by Faless** [comment]
That seems doable, I'll have a look into it as soon as I have some free time :+1:

**Comment by Malkverbena** [comment]
Hello again!
Is there any possibility to add channels to RPC?
I also would like to undestand better how it works.
On the code, I'd track the networking from node till scenetree, after it things became confusing.
I don't undestand how the packetpeer works.
Is there any tutorial or online information explaning how it works internally?
Could you give some tip of how networkingpeer works?

Regards!

**Comment by Faless** [comment]
Hi there, I'm sorry, I had very little time to look into this issue lately and I'm actually largely rethinking how this should be achieved, especially after godotengine/godot/pull/17227 is finished and merged.

**Comment by Malkverbena** [comment]
Oh, A lot of changes. 
Is there any impediment to using the channels in the multiplayer APIs?
It's a very useful resource, especially when you expect several connections.

**Comment by Faless** [comment]
> Is there any impediment to using the channels in the multiplayer APIs?

No, it should actually make things easier, as I won't have to reimplement all the code but just override few methods.

**Comment by Malkverbena** [comment]
Is there any plan to implement channels in the RPC method?
Imagine handle several worlds with several peers in each world.
Having channels available to handle such kind of server is very lacking.
This feature certainly would bring Godot closer to the term 'massive multiplayer'.
